image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -139,6 +139,10 @@ collector:
     enabled: null # bool
   runtimeConfig:
     enabled: null # bool
+    networking:
+      externalIps:
+        enable: null # bool
+      perContainerRateLimit: 1024 # int
```

Comment:
The config-shape is not for setting defaults, this is done in other files: `image/templates/helm/stackrox-secured-cluster/internal/defaults/...`.

Reply:
I set `perContainerRateLimit` to null here and to 1024 in the file that you mentioned.